src/services/websockets.js

```diff
@@ -1,7 +1,7 @@
 import * as SearchResultActions from '../actions/search-results.js';
 import * as TagActions from '../actions/tags.js';
 import Primus from '../../src/services/primus.js';
-const socketUrl = 'http://eb-ci.wmm63vqska.eu-west-1.elasticbeanstalk.com';
+const socketUrl = 'http://eb-ci.wmm63vqska.eu-west-1.elasticbeanstalk.com?auto_room=false';
```

Comment:
@nelsonic @jruts Not sure what a `Tracy` is, or what impact it might have. THis is taking advantage of [this code](https://github.com/numo-labs/graphql-backchannel/blob/master/app.js#L50-L51) in backchannel.